mycpp/mylib.h

```diff
@@ -387,6 +372,9 @@ class StrIter {
 };
 
 // TODO: Rewrite without vector<>, so we don't depend on libstdc++.
+//
+// TODO(Jesse): I like the sound of getting rid of std:vector
```

Comment:
Yes the old `mylib.h` isn't garbage collected, so it just uses `std::vector`.  The new one will be garbage-collected so we can't use it.  That's in `gc_heap.h` / `mylib2.h`

Reply:
Ahhh, got it.